.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -11,16 +11,16 @@ body:
     id: problem
     attributes:
       label: Issue Description
-      description: Please provide a description of the issue shown in the reproducible example.
+      description: Describe the issue.
     validations:
       required: true
   - type: textarea
     id: example
     attributes:
       label: Minimal Reproducible Example
       description: >
-        Please follow [this guide](https://matthewrocklin.com/minimal-bug-reports) on how to
-        provide a minimal, copy-pastable example.
+        Please provide a minimal, self-contained example.
+        See [Minimal Bug Reports](https://matthewrocklin.com/minimal-bug-reports) for guidance on how to write helpful examples.
```

Comment:
I don't think this reads as nicely anymore once rendered in Markdown,

> See Minimal Bug Reports for guidance on how to write helpful examples.

So in that sense, I prefer the previous phrasing over the proposed phrasing.

How about "Read [this article on Minimal Bug Reports](https://matthewrocklin.com/minimal-bug-reports) for guidance on how to do so"?

Reply:
Yes good point. I thought it would be helpful to emphasise that following the guide is helpful for us as maintainers, as I've seen a fair number of poorly written MREs recently.

I want to emphasise that the Guide is useful for writing _good_ MREs, so worth reading even if users already know how to copy & paste code examples. I'll try rephrasing it to be more explicit..